pycytominer/cyto_utils/load.py

```diff
@@ -32,23 +32,20 @@ def is_path_a_parquet_file(file: Union[str, pathlib.Path]) -> bool:
     """
     # type checking
     if not isinstance(file, (str, pathlib.Path)):
-        raise TypeError(f"file must be a str not {type(file)}")
+        raise TypeError(f"file must be a str or pathlib.Path not {type(file)}")
 
-    # Convert str to pathlib.Path object
+    # Convert str to pathlib.Path object and absolute path
+    # check if the file also exists while converting to absolute path
     if isinstance(file, str):
-        file = pathlib.Path(file)
-
-    # Convert to absolute path if necessary
-    if not file.is_absolute():
-        file = file.absolute()
+        try:
+            file = pathlib.Path(file).resolve(strict=True)
+        except FileExistsError as e:
+            raise FileNotFoundError(f"{str(file)} does not exist") from e
```

Comment:
A `FileExistsError` exception might not be triggered with the `try` step above (which attempts to resolve a path object but does not create it). Consider relying on `pathlib.Path.resolve(strict=True)` without this try/except block, relying on its built-in exception handling instead.